basecamp.go

```diff
@@ -193,6 +202,10 @@ func (c *Client) get(url string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	fmt.Println("BASECAMP REQ", url, "time", time.Since(start))
```

Comment:
Do we need this message here? If so, can we use `log` instead?

Reply:
No, we don't. It's just was backported from `pipes-api`